recipes/torchaudio/recipe.yaml

```diff
@@ -0,0 +1,180 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+
+context:
+  name: torchaudio
+  version: "2.4.1"
+  build_number: 0
+  cuda_version: ${{ env.get("CONDA_OVERRIDE_CUDA", default="None") }}
+  cuda: ${{ "true" if cuda_version != "None" else "false" }}
+  cuda_build_string: cuda_${{ cuda_version | version_to_buildstring }}
+  string_prefix: ${{ cuda_build_string if cuda ==  "true" else "cpu_" }}
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  - url: https://github.com/pytorch/audio/archive/refs/tags/v${{ version }}.tar.gz
+    sha256: a6b7e1b606ae353b9c4db1c8e754263f53f0457e9ab538e9669c0af0c46a0d5c
+    patches:
+      - patches/0001-point-to-correct-prefix.patch
+      - patches/3837.patch
+
+build:
+  number: ${{ build_number }}
+  skip:
+    - win
+    - match(cuda_version, "!=12")
+    - match(python, "!=3.11")
+  string: ${{ string_prefix }}py${{ python | version_to_buildstring }}h${{ hash }}_${{ build_number }}
+  variant:
+    use_keys:
+      # use cuda from the variant config, e.g. to build multiple CUDA variants
+      - ${{ "cuda" if cuda == "true" }}
+    # this will down-prioritize the cuda variant versus other variants of the package
+    down_prioritize_variant: ${{ 1 if cuda == "true" else 0 }}
+  script:
+    file: build
+    env:
+      cuda_compiler_version: ${{ cuda_version | default('None') }}
+
+requirements:
+  build:
+    - if: build_platform != target_platform
+      then:
+        - python
+        - cross-python_${{ target_platform }}
+        - pytorch
```

Comment:
I added pytorch in the build section during my early tests but I know some recipes only has it in the host section such as here: https://github.com/conda-forge/xformers-feedstock/blob/main/recipe/meta.yaml#L50

Maybe try to remove it?

Reply:
All good - fixed it by passing through CMAKE_ARGS :)